zcash_client_backend/CHANGELOG.md

```diff
@@ -31,7 +31,7 @@ and this library adheres to Rust's notion of
   - The `account` property of `WalletShieldedSpend` and `WalletShieldedOutput`
     now has the type `AccountId`.
 - `zcash_client_backend::welding_rig`:
-  - `scan_block` now takes `&[(AccountId, IncomingViewingKey)]`, instead of a
+  - `scan_block` now takes `&[(AccountId, SaplingIvk)]`, instead of a
```

Comment:
```suggestion
  - `scan_block` now takes `&[(AccountId, K: ScanningKey)]`, instead of a
```

We also need to document the added `ScanningKey` trait.